data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	This points to a vast intelligence network; it seems Hai aren't as naive as their cute furry exterior implies. You look up at the functionary, and she gives you a friendly little smile that seems more sinister now than it did a minute ago.`
+			`	Of course, if their intelligence agencies are so capable, shouldn't they have been able to solve their pirate problems?`
+				goto reading
+				to display
+					not "Hai Reveal [A11] Sound the Alarm: pirates book"
+			`	It seems that, although the Hai have trouble tracking small, independent, anarchist, groups, they have no trouble with large, established, hierarchies. Their very approach to anarchists was flawed: make deals with the biggest groups, as if those were heads of an anarchist hierarchy. The weakness of the Hai intelligence service is asymmetric warfare.`
+				goto reading
+				to display
+					has "Hai Reveal [A11] Sound the Alarm: pirates book"
+
+			label pirates
+			action
+				log `The Hai intelligence agencies' approach to pirates was to befriend the large anarchist groups, as if they were the head of an "anarchist hierarchy. This suggests they lack understanding of raw chaos; in essence, their greatest weakness is asymmetric warfare.`
+				set "Hai Reveal [A11] Sound the Alarm: pirates book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	The report begins with a brief summary of Hai involvement with anarchist groups, which dates back from the founding of the first anarchist colony, Freedom, around the same time as human first contact. Apparently, the Hai have had generally positive relations with the largest organizations on anarchist planets. The Hai trade humanitarian supplies or other goods for assistance from the anarchist planets, even ones as distant as Thule and Greenrock.`
+			`	Much of the nature of the "assistance from anarchist planets" is redacted, but it seems it mostly involves shutting down groups that try to reveal or harm the Hai. However, there are notes about deals with anarchists who oppose slavery, arranging massive rescues, bringing thousands of slaves at a time to freedom in Hai space. Also, there is an amnesty program for pirates who have committed no serious crimes, which explains the abundance of reformed pirates on Allhome.`
+			`	Recent anti-pirate actions by the Free Worlds and Republic Navy have weakened the large anarchist organizations that had deals with the Hai. This hindered the Hai ability to control pirates' activities near Ultima Thule, leading to the current crisis. Previous approaches that worked with large organizations, haven't worked with small pirate groups.`
```

Comment:
Did it? The war happened so FW was saying anarchist had more power than ever, unlike the republic which is having a hard time defending itself

Reply:
No, this is post-war. The Free Worlds removed pirates from several systems by befriending the local anarchist groups and having them kick the pirates out. This has reduced piracy in the Free Worlds to the lowest level in quite a few years. Hai Reveal A14 informs you that the Republic is about to do the same. That's what the paragraph refers to.

Reply:
Then mb focus on that part; say it was those FW pirates?

Reply:
Oh! I understand my mistake now. Yes, the Republic hasn't done that yet; they let pirates run rampant. I have corrected that. How does it look now?